ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -783,7 +767,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           </div>
         </WorkspaceEditSection>
         <WorkspaceEditSection header='Request a review of your research purpose for potential
-                                      stigmatization of research participants' required>
```

Comment:
There is no reason to signal to the user that this is Required any more.  If this boolean is somehow not a Yes or a No, that's a code error, and the user can't do anything about it.